src/script/ui/action/rdkitjstools.js

```diff
@@ -34,5 +34,13 @@ export default {
 		},
 		className: 'rdkitjs',
 		disabled: () => true
+	},
+	explicitHydrogen: {
+		title: 'Add/Remove Explicit Hydrogen',
```

Comment:
not sure if we need a shortcut field for the tool button

Reply:
I don't think we do